docs/content/docs/test/FlakyAnnotation.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Flaky Annotation
+type: docs
+---
+
+# `@Flaky` Annotation
+
+## What is flaky test
+
+A flaky test is a test that could fail or pass without project changes.\
+One could say this is unstable tests that fail sometimes but not every launch.\
+It happens when: 
+- test relies on an external environment: back-end, database, environment variables etc.
+- test had written incorrect. Mistakes easy to do in async jobs.
+- test framework has bugs
+
+Flaky tests make hard:
+- find real failures
+- trust to test failures
+- to react on failures as soon as possible \
+
+That's why we introduce annotion to mark your test flaky. 
+
+We want:
+- Minimize flakiness
+- Launch as many tests as we can on pull request checks
+- Help to fix common flaky problems
+
+## Annotation
+
+Such tests are marked with special annotation - `@Flaky`. You also able to provide an optional reason of flakiness for this test.
+
+```kotlin
+@Flaky(reason = "Relies on real back-end")
+class MyAwesomeTests {
+    // ...
+}
+```
+
+This annotation can be added both to the whole class and to a separate test:
+
+```kotlin
+class MyAwesomeTests {
+    @Flaky
+    @Test
+    fun foobar() {
+        // ...
+    }
+}
+```
+
+## Suppress `@Flaky` test failures
```

Comment:
It's not clear now how will change test behavior on CI after @Flaky will be added
We can describe current behavior that all Flaky don't run on PRs

Reply:
This is described in `test/Run.md`. You can find all information about running tests in this document.

Reply:
I think when you give any advice to use something you need to declare Rules
You said that developers must use `Flaky` but didn't answer all question they would have
I prefer to see all answers or all links to answers in one place with usage algorithm

Reply:
This doc was just renamed. In this PR I didn't have an intention to change it.